src/getColor.ts

```diff
@@ -2,16 +2,16 @@ import * as vscode from 'vscode';
 
 type Config = {
     regex: string;
-    color: string
+    color: string;
 }
 
-const getConfig = () => {
+export const getSetting = (value: string) => {
```

Comment:
the filename `getColor` doesn't fit anymore once you've exported `getSetting`
Maybe move this to a new `settings.ts` file